czech/showthread.lang.php

```diff
@@ -48,6 +50,7 @@
 $l['poll_closed'] = "Hlasování je ukončeno.";
 $l['poll_closes'] = "Hlasování bude ukončeno: {1}";
 $l['already_voted'] = "Již jste hlasoval(a).";
+$l['no_voting_permission'] = "Nemáte dostatečná práva, abyste mohl hlasovat v této anketě.";
```

Comment:
Mozna opravneni

Reply:
Tohle bylo trošku tricky (obtižné). Také jsem pro oprávnění. Bylo potřeba to upravit na více místech.
Více v commitu:
https://github.com/Eldenroot/MyBB_CZE-translation-1.8.x/pull/37/commits/175f0e3af28c0631d2d6b5865ef198602a4f1f6d